contrib/devtools/update-translations.py

```diff
@@ -160,7 +165,7 @@ def postprocess_translations(reduce_diff_hacks=False):
                     if translation is None:
                         continue
                     errors = []
-                    valid = check_format_specifiers(source, translation, errors, numerus)
+                    valid = check_format_specifiers(source, translation, errors, numerus) and not contains_bitcoin_addr(translation)
```

Comment:
Would like to have a report of what error occurred. Current code reports only what ends up in `errors` which is not set in this case, so the failure is silent.